ostorlab_insecure_flutter_app/lib/bugs/oracle_padding.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:convert';
+import 'package:encrypt/encrypt.dart';
+import 'package:ostorlab_insecure_flutter_app/bug_rule.dart';
+
+
+abstract class BugRule {
+  late BuildContext context;
+
+    String get _tag => 'OraclePadding';
+
+    String get description => 'This bug rule uses CBC insecure encryption mode.';
+
+    Future<void> run() async {
+
+        String data = "Jan van Eyck was here 1434";
+        String password = "ThisIs128bitSize";
+
+        final plainText = utf8.encode(data);
+        // specify an ecryption key and a random initialization vector
+        final key = Key.fromUtf8(password);
+        final iv = IV.fromLength(16);
+        // Encrypter instantiation with the CBC mode.
+        final encrypter = Encrypter(AES(key, mode: AESMode.cbc));
```

Comment:
```suggestion
        final encrypter = Encrypter(AES(key, mode: AESMode.cbc, padding: 'PKCS7'));
```
CBC algorithm itself is not inherently vulnerable to Oracle padding attacks, it is the padding scheme that is vulnerable.

Reply:
but the PKCS7 scheme is not the only padding scheme that is vulnerable to the oracle padding attack. both PKCS5 and ISO 10126 are also vulnerable. and also we currently raise this as info so it wouldnt hurt.

Reply:
good point!